LondonBritishLibrary/orient/BLorient547.xml

```diff
@@ -107,13 +89,204 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                             <citedRange unit="page">90b-91a</citedRange>
                                         </bibl>
                                     </listBibl>
-                                    
                                 </source>
                             </recordHist>
                         </adminInfo>
                     </additional>
+
+
+<msPart xml:id="p1">
+<msIdentifier></msIdentifier>
+
+<msContents>
+
+<msItem xml:id="p1_i1">
+<locus from="2r"/>
+<title xml:lang="en" type="incomplete">Confession of faith</title>
+<note>Incomplete at the beginning.</note>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i2">
+<locus from="3r"/>
+<title xml:lang="en" type="complete">A hymn consisting of verses from various psalms</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+</msContents>
+
+<physDesc>
+
+    <handDesc>
+    <handNote script="Ethiopic" xml:id="h1"></handNote>
+    </handDesc>
+
+</physDesc>
+
+</msPart>
+
+
+<msPart xml:id="p2">
+<msIdentifier></msIdentifier>
+
+<msContents>
+
+<msItem xml:id="p1_i2.1">
+<locus from="4r"/>
+<title xml:lang="en" type="complete" ref="LIT1960Mashaf">The missal</title>
+<notatedMusic><desc>Musical notes have been added.</desc></notatedMusic>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i2.1.1">
+<locus from="4r"/>
+<title xml:lang="en" type="complete" ref="LIT1960Mashaf#Daily">Daily prayers</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i2.1.2">
+<locus from="28r"/>
+<title xml:lang="en" type="complete" ref="LIT1960Mashaf#SerataQeddase">The ordinary Canon of the Mass</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="p1_i2.1.3">
+<locus from="68r"/>
+<title xml:lang="en" type="complete" ref="LIT3189Anapho">Anaphoras</title>
```

Comment:
LIT1960Mashaf#Anaphoras, if it is a collection, it is much better to use the second-level ids

Reply:
This is applicable for all the following msItems